### ISSUE TITLE
Console Output Not Human Readable

### ISSUE THREAD

**Issue by jessiewestlake** [FabricMC/fabric#1111]
I have a server running on AWS Linux. Everything works fine, but when issuing commands through the console or through RCON, it almost always comes back with a pointer to some message, but not an actual message. Running most of the same commands in the chat window while logged on as a opped player, they return info fine.

Minecraft v1.13.1

Can't remember how to determine which fabric server version is installed, but it was the version current toward the end of 1.13.1.

````
[user@localhost]$ /weather clear
commands.weather.set.clear

[user@localhost]$ /help enchant
/enchant <targets> <enchantment> [<level>]

[user@localhost]$ /banlist
commands.banlist.none

[user@localhost]$ /list
commands.list.players
````

In the chat window in-game:

````
/list
There are 1 of a max of 20 players online: R3alTim3
````

**Comment by i509VCB** [comment]
We will probably need the version you are running in this case. Fabric loader should print that out at the beginning of the server start process.

**Comment by i509VCB** [comment]
Closing due to lack of feedback. If this is still an issue please open a new issue.

**Comment by XNarno** [comment]
Hello, I have a dedicated server at home which run on Ubuntu server 22.04.1 LTS.
I'm running a Fabric server in 1.18.2 with the Fabric Minecraft Server Launcher and this is my start command :
`java -Xmx4G -jar fabric-server-mc.1.18.2-loader.0.14.9-launcher.0.11.1.jar nogui`

I have the same issue, I have these types of returns in my console : 


`>` weather clear
[15:08:30 INFO] [Server thread]: [DedicatedServer] commands.weather.set.clear

`>` say hello
[15:09:18 INFO] [Server thread]: [DedicatedServer] chat.type.announcement

`>` list
[15:12:20 INFO] [Server thread]: [DedicatedServer] commands.list.players


I thought "Better Fabric Console" mod will resolve my issue but no, it's just more beautiful, so I don't know what to do.
I tried to run my server not with the Minecraft Server Launcher but with a classic installation, but it's not working.
I also tried to put Fabric Loader 0.14.10 but nothing more.

This is my mod list : [https://pastebin.com/qsGt1Z5y](url)
